utils/nctl/sh/scenarios/chainspecs/itst14.chainspec.toml.in

```diff
@@ -13,7 +13,7 @@ hard_reset = false
 # If it is an integer, it represents an era ID, meaning the protocol version becomes active at the start of this era.
 activation_point = '${TIMESTAMP}'
 # The era ID starting at which the new Merkle tree-based hashing scheme is applied.
-merkle_tree_hash_activation = 2,
+verifiable_chunked_hash_activation = 2,
```

Comment:
I know that it's not a part of your changes, but will it parse correctly with the `,` at the end?

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2598/commits/a0f6e0396fc3300a106b7af741ef127c21aeea47.